tests/wpt/metadata/fullscreen/api/document-fullscreen-enabled.html.ini

```diff
@@ -0,0 +1,5 @@
+[document-fullscreen-enabled.html]
+  type: testharness
+  [Document.fullscreenEnabled]
+    expected: FAIL
```

Comment:
I don't get your point. `contentDocument` is implemented and `fullscreenEnabled` is also implemented, so why is it still failing?